docs/pull_request_template.md

```diff
@@ -0,0 +1,47 @@
+<!--
+Make sure to familiarise yourself with the Runtime release process at https://github.com/polkadot-fellows/runtimes?tab=readme-ov-file#release-process 
+and the Runtime release guidelines at https://github.com/polkadot-fellows/runtimes?tab=readme-ov-file#release-guidelines.
+Use this PR template to streamline the release process, communicate on the changes that you are proposing, and provide a clear course of action for network stakeholders.
+-->
+
+## I'm submitting...
+<!---
+REQUIRED:
+Classify the type of PR your are submitting.
+-->
+  - [ ] a priority PR (i.e fixes and patches)
+  - [ ] a recommended PR (i.e enhancements, backports, migrations)
+  - [ ] an additional PR (i.e small configurations, simple migrations, general cleanup)
+
+
+### What kind of changes does this PR introduce?
+<!---
+OPTIONAL:
+Indicate if merging this PR will result in breaking changes (i.e changes to transaction/event/error encoding, polkadot-sdk migrations, or XCM and storage format), disruptions, 
+or downtimes for the network. When submitting a breaking change, please make sure to ping @SBalaguer and @anaelleltd so that they can notify ecosystem teams and builders.
+Otherwise, simply delete this section.
+-->
+
+
+### Who will be impacted by theses changes in practice?
+<!---
+OPTIONAL:
+Indicate all network stakeholders (i.e teams working on parachains, wallets, UIs, CEXes, or DEXes) that this PR will affect.
+Otherwise, simply delete this section.
+-->
+
+
+### How should impacted team/builders follow up on these changes?
+<!---
+OPTIONAL:
+Recommend a course of action (i.e resources for code refactoring or further reading) for network stakeholders.
```

Comment:
This should just be implicit in the prior section.

Reply:
> This should just be implicit in the prior section.

Sure thing, but this is a new and important change to established procedures. There should be no ambiguity about the type of information that is required from code contributors.

Reply:
Sure, why does it need a new section? Just put it as an expectation in the prior section.

Reply:
> Sure, why does it need a new section? Just put it as an expectation in the prior section.

Thanks for clarifying. This has been done.